lib/tasks/job_fetch.rake

```diff
@@ -1,5 +1,5 @@
 namespace :job_fetch do
-  desc "Run the stack overflow fetcher"
+  desc "Run all job fetchers"
 
   task all_jobs: :environment do
```

Comment:
To keep this DRY - this should probably call the other tasks directly:

Check out some ways to do that here: http://stackoverflow.com/questions/577944/how-to-run-rake-tasks-from-within-rake-tasks